src/docs/release/breaking-changes/test-widgets-flutter-binding-clock.md

```diff
@@ -0,0 +1,104 @@
+---
+title: `TestWidgetsFlutterBinding.clock` change
+description: The `Clock` implementation will now come from package:clock.
+---
+
+## Summary
+
+The `TestWidgetsFlutterBinding.clock` will now come from `package:clock` and
+not `package:quiver`.
+
+## Context
```

Comment:
somewhere you should give the error message that people will see, so this guide comes up in Google searches